tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+from typing import Iterator
+
+import pytest
+from _pytest.tmpdir import TempPathFactory
+
+CONTENT = (
+    "Typer is a library for building CLI applications that users will love using and developers "
+    "will love creating. Typer is a library for building CLI applications that users will love using and "
+    "developers will love creating. Based on Python type hints. "
+    "It's also a command line tool to run scripts, automatically converting them to CLI applications."
+)
+
+
+@pytest.fixture(scope="session")
+def sample_textfile(tmp_path_factory: TempPathFactory) -> Iterator[Path]:
```

Comment:
I believe you can use just tempdir https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture . The typing should be Path